### ISSUE TITLE
Add configuration to turn off linting features

### ISSUE THREAD

**Issue by sr-verde** [shinagawa-web/gomarklint#16]
Hey, 
thanks for your tool. I would love to have the possibility to turn off single linting features to e.g. turn off checking heading level consistency.

Best

**Comment by shinagawa-web** [comment]
@sr-verde 
Thanks a lot for your feedback — and I'm happy to let you know that this feature is now available! 🎉

As of **v1.0.3**, you can disable the heading-level consistency check using the new flag:

```bash
gomarklint --enable-heading-level-check=false
```

It also works via the config file (`.gomarklint.json`):

```json
{
  "enableHeadingLevelCheck": false
}
```

Please give it a try and let me know if it works as expected.
Thanks again for helping improve gomarklint!
